x/ecocredit/simulation/genesis.go

```diff
@@ -109,7 +109,6 @@ func RandomizedGenState(simState *module.SimulationState) {
 		CreditClassFee:       creditClassFee,
 		AllowedClassCreators: allowedClassCreators,
 		AllowlistEnabled:     allowListEnabled,
-		CreditTypes:          creditTypes,
```

Comment:
Without credit-type sims will fail. We should initialize `CreditTypeTable` with least one credit-type.

Reply:
https://github.com/regen-network/regen-ledger/pull/1043/commits/467f7da6e45ebb95aeeaadf29b81274f2ef0199b